tests/file_uploads/tests.py

```diff
@@ -38,6 +38,8 @@
     '../..\\hax0rd.txt',        # Relative path, mixed.
     '..&#x2F;hax0rd.txt',       # HTML entities.
     '..&sol;hax0rd.txt',        # HTML entities.
+    'hax0\x00rd\x00.txt\x00',   # Non-printable - null characters.
+    'hax0r\nd.txt',             # Non-printable.
```

Comment:
We shouldn't mix file names that are candidates for path traversal with sanitizing non-printable chars. I moved it to a separate test method.